admin/src/user/list.tsx

```diff
@@ -18,6 +13,13 @@ export const UserList = () => (
       <TextField source="id" />
       <TextField source="username" />
       <EmailField source="email" />
+      <CustomManyCount
```

Comment:
not fan of the name, I don't understand what this component will display/act on

Reply:
This will be resolved in a future PR (I will need to have a custom join table and this component will be removed).